pkg/config/config_test.go

```diff
@@ -319,23 +319,38 @@ func (s *MySuite) TestNewBlueprint(c *C) {
 	dc := getDeploymentConfigForTest()
 	outFile := filepath.Join(tmpTestDir, "out_TestNewBlueprint.yaml")
 	dc.ExportBlueprint(outFile)
-	newDC := NewDeploymentConfig(outFile)
+	newDC, _ := NewDeploymentConfig(outFile)
 	c.Assert(dc.Config, DeepEquals, newDC.Config)
 }
 
 func (s *MySuite) TestImportBlueprint(c *C) {
-	obtainedBlueprint := importBlueprint(simpleYamlFilename)
+	obtainedBlueprint, _ := importBlueprint(simpleYamlFilename)
 	c.Assert(obtainedBlueprint.BlueprintName,
 		Equals, expectedSimpleBlueprint.BlueprintName)
 	c.Assert(
-		len(obtainedBlueprint.Vars["labels"].(map[interface{}]interface{})),
+		len(obtainedBlueprint.Vars["labels"].(map[string]interface{})),
 		Equals,
 		len(expectedSimpleBlueprint.Vars["labels"].(map[string]interface{})),
 	)
 	c.Assert(obtainedBlueprint.DeploymentGroups[0].Modules[0].ID,
 		Equals, expectedSimpleBlueprint.DeploymentGroups[0].Modules[0].ID)
 }
 
+func (s *MySuite) TestImportBlueprintExtraFieldThrowsError(c *C) {
```

Comment:
nit: Naming of variations of a test function should be separated by an underscore, as it's done elsewhere:
https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/pkg/modulereader/resreader_test.go#L134

So this should be `TestImportBlueprint_ExtraFieldThrowsError`

Reply:
Good call. I actually went with `TestImportBlueprint_ExtraField_ThrowsError` in the pattern of `<method>_<state>_<outcome>`